sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/InfiniteRecursion.cs

```diff
@@ -90,7 +90,7 @@ string Prop0
 
         object Prop1
         {
-             get // Noncompliant {{Add a way to break out of this property accessor's recursion.}}
+            get
             {
                 return (new InfiniteRecursion())?.Prop1;
```

Comment:
you could replace this instantiation with adding a field `InfiniteRecursion instanceField` and call `return instanceField?.Prop1` ?

Reply:
I added a test with a field access, but I kept this instanciation for reference